src/overlays/actors/ovl_En_Encount1/z_en_encount1.c

```diff
@@ -51,6 +51,8 @@ void EnEncount1_Init(Actor* thisx, PlayState* play) {
     spawnRange = 120.0f + (40.0f * this->actor.world.rot.z);
     this->spawnRange = spawnRange;
 
+    if (1) {}
+
     PRINTF("\n\n");
```

Comment:
kinda curious if this matches as `if(OOT_DEBUG)` surrounding the printfs
(if it doesn not sure if we'd want that)

Reply:
It does (for the record, `if (1)` surrounding the printfs matches too)